apps/web/utils/actions.ts

```diff
@@ -89,6 +92,22 @@ export async function deleteAccountAction() {
   const session = await auth();
   if (!session?.user.email) throw new Error("Not logged in");
 
+  const user = await prisma.user.findUniqueOrThrow({
+    where: { id: session.user.id },
+    select: {
+      premium: {
+        select: {
+          id: true,
+          tier: true,
+          lemonSqueezySubscriptionItemId: true,
```

Comment:
I don't think we need any of these.

We need: `lemonSqueezySubscriptionId` (which is different to `lemonSqueezySubscriptionItemId`)

Reply:
Done